root/area/edit_form.tt

```diff
@@ -42,6 +42,16 @@
 
 <script type="text/javascript">//<![CDATA[
   (function () {
+    const wasEnded = $('#id-edit-area\\\.period\\\.ended').prop('checked');
```

Comment:
Yeah, it looks we can reduce some duplication there as well. Just some of the strings differ.